.github/workflows/build_linux_app.yml

```diff
@@ -0,0 +1,21 @@
+name: Build_Linux_app
+
+on:
+  push:
+    tags:
+    - "*"
+
+jobs:
+  Build_Linux_app:
```

Comment:
It's an [ELF binary](https://en.wikipedia.org/wiki/Executable_and_Linkable_Format) that we build, isn't it?

Would you mind calling this `build_linux_elf`. To avoid confusion with the [AppImage](https://appimage.org/) format that we might also want to build.

Also affects line 1 of this file and the file name itself, by the way.